tests/ui/openid_connect_mock.rs

```diff
@@ -74,13 +77,65 @@ type CustomIdTokenFields = IdTokenFields<
 type CustomTokenResponse = StandardTokenResponse<CustomIdTokenFields, CoreTokenType>;
 // end cascade
 
+// This failure mode can be invoked for a user at user creation time.
+// The failure mode will be activated on a 'refresh_token' request
+// from krill to the OIDC Provider
+#[derive(Clone, Debug)]
+enum RefreshTokenFailureMode {
+    // These are the RFC 6749 5.2 Errors
+    InvalidRequestErrorResponse,
+    InvalidClientErrorResponse,
+    InvalidGrantErrorResponse,
+    InvalidScopeErrorResponse,
+    UnauthorizedClientErrorResponse,
+    UnsupportedGrantTypeErrorResponse,
+    // These various generic failure modes,
```

Comment:
I'm not sure I understand this comment, I would expect "these ... <insert verb>" but the verb missing missing. What were you trying to say here?

Reply:
These errors are not mentioned in Section 5.2, They are error codes that originate from the generic network or server-side problems.